argo/template/resolve-lock.yaml

```diff
@@ -0,0 +1,62 @@
+metadata:
+  name: tipocket-resolve-lock
+  namespace: argo
+spec:
+  templates:
+    - name: tipocket-resolve-lock
+      inputs:
+        parameters:
+          - name: ns
+            default: tipocket-resolve-lock
+          - name: purge
+            default: "true"
+          - name: repository
+            default: pingcap
+          - name: image_version
+            default: nightly
+          - name: storage_class
+            default: standard
+          - name: nemesis
+            default: "shuffle-leader-scheduler,shuffle-region-scheduler,random-merge-scheduler,scaling"
+          - name: run_time
+            default: "6h"
+          - name: loki-addr
+            default: http://gateway.loki.svc
+          - name: loki-username
+            default: loki
+          - name: loki-password
+            default: admin
+          - name: tidb_config
+            default: ""
+          - name: tikv_config
+            default: ""
+          - name: pd_config
+            default: ""
+          - name: tikv-replicas
+            default: "3"
+      metadata:
+        labels:
+          ns: "{{inputs.parameters.ns}}"
+      container:
+        name: tipocket
+        image: 'mahjonp/tipocket:green_gc'
```

Comment:
revert pingcap/tipocket

And could you update template and workflow, we have updated them many times since this PR.

Reply:
Ahh, this case needs a special image that enables failpoints in TiDB.

Reply:
> Ahh, this case needs a special image that enables failpoints in TiDB.

Perhaps you can ask @zyguan for help, they may have the CI job

Reply:
You can use `hub.pingcap.net/qa/{tidb,tikv,pd}:{master,release-4.0}-failpoint` directly, they are built daily.

However, since a special env var needed to be set for enabling the failpoint, we need to use a custom startup script. see https://github.com/pingcap/automated-tests/blob/dev/pkg/resource/impl/k8s/assets/start-tidb.sh#L40

Reply:
@zyguan does tidb-server support to enable failpoint in runtime? For example activate some failpoints through HTTP endpoint?

Reply:
Failpoint itself serves on [`GO_FAILPOINTS_HTTP`](https://github.com/pingcap/failpoint/blob/b230c36c413c4574ab97d09419f7e796eec4e565/failpoints.go#L76-L81), it also requires specifying a env var when start tidb. 

Besides, the `enableTestAPI` is special, one should enable it before tidb setup status service. So, AFIK, there is no workaround currently.